app/main.py

```diff
@@ -1,3 +1,65 @@
+from __future__ import annotations
+from typing import Any
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(
+            self,
+            km: int
+    ) -> None:
+        self.km = km
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __add__(self, other: Distance | int) -> Distance:
+        if isinstance(other, Distance):
+            return Distance(
+                self.km + other.km
+            )
+        else:
+            return Distance(self.km + other)
+
+    def __iadd__(self, other: Distance | int) -> Distance:
+        self.km = (self.__add__(other)).km
+
+        return self
+
+    def __mul__(self, other: int | float) -> Distance:
+        return Distance(self.km * other)
+
+    def __truediv__(self, other: int | float) -> Distance:
+        return Distance(
+            round(self.km / other, 2)
+        )
+
+    def __eq__(self, other: Any) -> bool:
+        if not isinstance(other, Distance):
+            return self.km == other
+        return self.km == other.km
+
+    def __lt__(self, other: Distance) -> bool:
+        if not isinstance(other, Distance):
+            return self.km < other
+        return self.km < other.km
+
+    def __gt__(self, other: Distance | (int, float)) -> bool:
```

Comment:
```suggestion
    def __gt__(self, other: Distance | int | float) -> bool:
```
it's not the right annotation. YOu suggesting that it could be either Distance or tuple